test/pool/TestPoolIntegration.spec.ts

```diff
@@ -912,6 +1140,1517 @@ contract("Pool", async (accounts) => {
+              expectedMaxTopup: expectedMaxTopup,
+              // ETH minSharingThreshold = 50 ETH, 74 ETH > 50 ETH = bigLoan = !IsSmall
+              expectedIsSmall: false,
+            });
+
+            const alreadyDeposited = await pool.balance(a.member1, ETH_ADDR);
+            await pool.methods["deposit()"]({
+              from: a.member1,
+              value: expectedMinTopup.sub(alreadyDeposited),
+            });
+
+            // topup
+            await pool.topup(a.user1, bETH_addr, expectedMinTopup, false, {
+              from: a.member1,
+            });
+
+            // - member deposit & do partial liquidate BAT
+            // -------------------------------------------
+            debtToLiquidate = await avatar1.getMaxLiquidationAmount.call(cETH_addr);
+            halfDebtToLiquidate = debtToLiquidate.div(new BN(2));
+            result = await avatar1.calcAmountToLiquidate.call(cETH_addr, halfDebtToLiquidate);
+            amtToDeductFromTopup = result["amtToDeductFromTopup"];
+            amtToRepayOnCompound = result["amtToRepayOnCompound"];
+
+            // member deposit
+            await pool.methods["deposit()"]({
+              from: a.member1,
+              value: amtToRepayOnCompound,
+            });
+
+            // partial liquidation
+            // -------------------
+            await pool.liquidateBorrow(a.user1, bBAT_addr, bETH_addr, halfDebtToLiquidate, {
+              from: a.member1,
+            });
+
+            // validate deposit & topup balance
+            expect(await pool.balance(a.member1, ETH_ADDR)).to.be.bignumber.equal(ZERO);
+            expect(await balance.current(pool.address)).to.be.bignumber.equal(ZERO);
+            expect(await pool.topupBalance(a.member1, ETH_ADDR)).to.be.bignumber.equal(
+              expectedMinTopup.div(new BN(2)),
+            );
+
+            // validate cBAT balances
+            // 10000 BAT = $10000
+            // 6975 ZRX = $6975
+            // Collateral = $16975
+            // Borrowed = 74 ETH @ 115 = $8510
+            // hence 50% closeFactor of $8510 = $4255 worth of BAT (4255 BAT) open for liquidation
+            // Liquidating half, hence $4255 / 2 => 2127.5 BAT liquidate
+            // liquidationIncentive 110% = 2127.5 * 110 / 100 = 2340.25 BAT underlyingLiquidated
+            const pointTwoFive = ONE_BAT.div(new BN(4));
+            const underlyingBATLiquidated = ONE_BAT.mul(new BN(2340)).add(pointTwoFive);
+            await validateShareReceived(underlyingBATLiquidated, cBAT_addr, a.member1);
+
+            expect(await avatar1.canLiquidate.call()).to.be.equal(true);
+            expect(await avatar1.canUntop.call()).to.be.equal(true);
+            expect(await avatar1.isPartiallyLiquidated()).to.be.equal(true);
+          });
+        });
+      });
+
+      describe("Full liquidation, then another liquiation for the same debt, then another liquiation for different debt", async () => {
```

Comment:
we need another test where the liquidation is partial for the first debt, and then another liquidation for the second debt.